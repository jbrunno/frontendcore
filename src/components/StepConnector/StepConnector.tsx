import React from 'react';
import {
  StepConnector as MuiStepConnector,
  StepConnectorProps as MuiStepConnectorProps,
  stepConnectorClasses as MuiStepConnectorClasses,
} from '@mui/material';

export const stepConnectorClasses = MuiStepConnectorClasses;
export type StepConnectorProps = MuiStepConnectorProps;

export function StepConnector(props: StepConnectorProps) {
  return <MuiStepConnector {...props} />;
}

export default StepConnector;
