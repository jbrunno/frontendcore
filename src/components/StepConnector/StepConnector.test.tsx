import React from 'react';
import { render } from '@test-utils/component-testing';
import { StepConnector } from './StepConnector';

test('renders the StepConnector', () => {
  const { container } = render(<StepConnector />);
  expect(container).toMatchSnapshot();
});
