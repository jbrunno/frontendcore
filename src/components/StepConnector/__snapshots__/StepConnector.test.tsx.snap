// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the StepConnector 1`] = `
.emotion-0 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
}

.emotion-1 {
  display: block;
  border-color: #bdbdbd;
  border-top-style: solid;
  border-top-width: 1px;
}

<div>
  <div
    class="MuiStepConnector-root MuiStepConnector-horizontal emotion-0"
  >
    <span
      class="MuiStepConnector-line MuiStepConnector-lineHorizontal emotion-1"
    />
  </div>
</div>
`;
